eng/pipelines/templates/jobs/aggregate-reports.yml

```diff
@@ -0,0 +1,34 @@
+resources:
+  repositories:
+    - repository: azure-sdk-tools
+      type: github
+      name: Azure/azure-sdk-tools
+      endpoint: azure
+    - repository: azure-sdk-build-tools
+      type: git
+      name: internal/azure-sdk-build-tools
+
+trigger: none
+pr: none
+
+jobs:
+- job: 'ValidateDependencies'
+  variables:
+  - template: ../variables/globals.yml
+
+  pool:
+    vmImage: 'windows-2019'
+
+  steps:
+  - template: ../steps/analyze.yml
```

Comment:
Doesn't this cause all the analyze steps to happen both here and in every pipeline? shouldn't there be some filtering of some sort?